DuplicatesFinder_v4/ViewModels/MainViewModel.cs

```diff
@@ -131,42 +127,101 @@ public ICommand OnClickSearch
                 {
                     DuplicatesViewModel.CollectionForDuplicatesView.Clear();
 
-                    if (ispics == false && isdocs == false && isvideos == false )
+                    if (ispics == false && isdocs == false && isvideos == false)

```

Comment:
make PropertyChanged as property
order properties and methods

Reply:
has done